medusa/indexers/indexer_api.py

```diff
@@ -18,7 +18,10 @@ def __del__(self):
     @PlexFallBackConfig
     def indexer(self, *args, **kwargs):
         if self.indexer_id:
-            return indexerConfig[self.indexer_id]['module'](*args, **kwargs)
+            indexer_obj = indexerConfig[self.indexer_id]['module'](*args, **kwargs)
+            indexer_obj.indexer = self.config['id']
```

Comment:
at least both are now using the indexer_config.
Meaning the indexer_config is leading.